theia-extensions/viewer-prototype/src/common/backend-service.ts

```diff
@@ -0,0 +1,8 @@
+import { CancellationToken } from '@theia/core';
+
+export const backendServicePath = '/services/backend-service';
+export const BackendService = Symbol('BackendService');
+
+export interface BackendService {
```

Comment:
Should the name be a bit clearer? It feels that name `BackendService` is too general

Reply:
Changed to BackendFileService